### PR TITLE
eslint: update ignore path to coverage results

### PR DESCRIPTION
**Pull request by jbergstroem**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 node_modules/
-tests/results/
-build/
+tests/unit/results/
+build/
```
Comment (NiGhTTraX) [inline .eslintignore]: Can you please add the new line back?

**Review by NiGhTTraX — CHANGES_REQUESTED** [review]
Thanks!

File: .eslintignore (hunk 1 above)
Comment (jbergstroem) [reply .eslintignore]: Apologies. Editors...

Comment (NiGhTTraX) [reply .eslintignore]: I know, right... Thanks again for your contributions!

**Pull request merged**